__tests__/app.test.js

```diff
@@ -27,4 +26,41 @@ describe("GET /api/topics", () => {
                 });
             });
     });
+
+    it("GEt /api/articles/:article_id :- should return :200 and array articles with that id", () => {
+        return request(app)
+            .get("/api/articles/3")
+            .expect(200)
+            .then(({ body }) => {
+                const articles = body.articles;
+
+                expect(articles).toHaveLength(1);
+                articles.forEach((articles) => {
```

Comment:
if you know that there's only one article here, it would be a more robust test to check against the specific title, topic, author etc of that article, not iterating through it